shared/src/business/entities/DocketEntry.ts

```diff
@@ -41,6 +41,29 @@ import {
   createISODateString,
 } from '../utilities/DateHandler';
 
+const canViewSTIN = (entry, petitionDocketEntry, user) => {
```

Comment:
We really only pulled this out into into its own function because eslint was complaining about the complexity of `isDownloadable`. The thinking was that it's essentially a private helper function containing some of the details of `isDownloadable` that are specific to the STIN document type.

Cody and I are leaning toward putting this back inline in `isDownloadable` and just silencing the eslint warning, but are curious to hear thoughts on this.

Reply:
That makes sense to me. I realize now that you did not export this constant, so it is just for this file. That makes sense and I take back what I said. I like the extraction, but my only ask would be if you are going to extract it, then type the function.

Reply:
Types, coming right up!